domain/src/main/kotlin/no/nav/su/se/bakover/domain/sak/SakRestans.kt

```diff
@@ -9,7 +9,7 @@ data class SakRestans(
     val behandlingsId: UUID,
     val restansType: RestansType,
     val status: RestansStatus,
-    val opprettet: Tidspunkt,
+    val behandlingStartet: Tidspunkt?,
```

Comment:
Litt dumt at denne blir nullable for alle variantene, selvom det kun gjelder for papirsøknader :/

Reply:
Gjelder vel vanlige søknader også?

Reply:
Men ja, kunne vurdert å splitte opp denne på type på et tidspunkt!